lms/djangoapps/course_home_api/dates/v1/views.py

```diff
@@ -48,6 +48,7 @@ class DatesTabView(RetrieveAPIView):
             link: (str) An absolute link to content related to the date event
                 (ex. verified link or link to assignment)
             title: (str) The title of the date event
+        has_ended: (bool) Indicates whether course has ended
         missed_deadlines: (bool) Indicates whether the user missed any graded content deadlines
         missed_gated_content: (bool) Indicates whether the user missed gated content
```

Comment:
Can you update the documentation here to better reflect the dates_banner_info object? It looks like it is sent as one object, and I think your documentation below in the outline API much better reflects what happens here.